src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -830,21 +835,7 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         }
         _interactivity.Initialize();
 
-        _AttachDxgiSwapChainToXaml(reinterpret_cast<HANDLE>(_core.SwapChainHandle()));

```

Comment:
3. Race condition in `GetSwapChainHandle()` due to it being called outside
of the console lock and with single-threaded Direct2D enabled

Reply:
(things that may cause device contention)

Reply:
`wtd "sp;sp;sp;sp;sp"` worked fine in a debug build. Since I tested this with an added `Sleep(3000)` I can't conceive any situation myself were this would fail to work. 🤔